### DISCUSSION TITLE
Next CAMNOR-core meeting

### DISCUSSION THREAD

**Discussion by DirkOlivie** [NorESMhub/CAM#67] [Team Posts]
Shall we shortly have a short camnor-core meeting, to discuss some recent developments?

There have been some changes to the CAM-repository :
1. added new branch cam-Nor-dev.  This is supposed to be the development branch, where post-CMIP6 modifications should be brought in : bug fixes, answer changing modifications, ...
2. added new branch cam_cesm2_1_rel_05-Nor_v1.0.2_keyClim-withoutanthraer.  This is supposed to be the branch used for the KeyCLIM natural aerosol-only experiment.
3. updated the README.md file in master branch (contains information on new branches)

Possible topics to discuss :
1. What updates can still go in the cam_cesm2_1_rel_05-Nor branch?  (e.g., covid-19 compsets, user_namelist, ...)
2. The KeyCLIM experiments which need developments in CAM-Nor, can they (all) have their own branch in NorESMhub/CAM?  Who can have write access in that repository?
3. Is cam-Nor-dev ok to be the new development branch?
4. CESM2.2.0 is released, and uses the CAM-tag cam_cesm2_2_rel_02.  Shall we (try to) update cam-Nor-dev to this tag?
5. How do we communicate modifications in the CAM-repository, e.g., the existence of cam-Nor-dev?
6. What naming convention for new upcoming branches shall we use?
7. Do all the README.md files in all branches are supposed to be the same?

There might be other issues.

Is there a possibility one of the mornings this week or next week?

**Comment by oyvindseland** [comment]
Not used for the CAM-Nor meetings now
